source/view.c

```diff
@@ -972,11 +1055,18 @@ static void rofi_view_refilter ( RofiViewState *state )
         tokenize_free ( state->tokens );
         state->tokens = NULL;
     }
+    if ( state->pattern ) {
```

Comment:
I find `state->tokens` hard to use in the revised fuzzy finding algorithm because it requires the complete input so I added `char* pattern` to `view-internal.h`